datasets/simulated.py

```diff
@@ -15,23 +15,23 @@ class Dataset(BaseDataset):
         "n_samples, n_features": [
             (100, 200),
         ],
-        "normalize": [True, False],
+        "standardize": [True, False],
     }
 
-    def __init__(self, n_samples=10, n_features=50, normalize=False):
+    def __init__(self, n_samples=10, n_features=50, standardize=False):
         # Store the parameters of the dataset
         self.n_samples = n_samples
         self.n_features = n_features
-        self.normalize = normalize
+        self.standardize = standardize
         self.random_state = 0
 
     def get_data(self):
 
         rng = np.random.RandomState(self.random_state)
         X = rng.randn(self.n_samples, self.n_features)
         y = rng.randn(self.n_samples)
-        if self.normalize:
-            X /= np.linalg.norm(X, axis=0)
+        if self.standardize:
+            X /= np.linalg.norm(X, axis=0) / np.sqrt(len(y))
```

Comment:
```suggestion
            X /= X.std(axis=0)
```

?

but this does a centering. At least that's what standardize suggests

Reply:
What I really need is ||X_j||^2 / n_samples = 1   (the lipschitz constant) must be 1 otherwise, prox stepsize * MCP can go havoc). `X /= X.std(axis=0)` does not enforce that.

You're right standardize is misleading. what do you suggest? `scale` ?